src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,143 @@
+// Header
+.header {
+  background-color: $color-white;
+  box-shadow: $box-shadow;
+
+  &__container {
+    margin: 0 $dim-main-margin;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__logo-link {
+    @extend %flex-center;
+  }
+
+  &__logo-img {
+    @extend %pointer;
+    width: #{$dim-logo-size};
+    height: #{$dim-logo-size};
+    margin: $dim-main-padding $dim-nav-gap $dim-main-padding 0;
+  }
+}
+
+// Navigation
+.nav {
+  display: flex;
+
+  &__burger-label {
+    @extend %flex-center;
+    @extend %pointer;
+    transform: rotate(90deg);
+    font-size: $dim-fontsize-big;
+    line-height: $dim-fontsize-big;
+    font-weight: bold;
+    z-index: 3;
+  }
+
+  &__burger-checkbox {
+    display: none;
+  }
+
+  &__burger-checkbox:checked ~ .nav__burger-label {
+    position: fixed;
+    right: $dim-main-margin + $dim-main-padding;
+    top: $dim-main-padding + 4px;
+  }
+
+  &__list {
+    display: none;
+    @extend %flex-center;
+    gap: $dim-nav-gap;
+    position: fixed;
+    width: 100%;
+    min-height: 100vh;
+    top: 0;
+    left: 0;
+    flex-direction: column;
+    list-style: none;
+    background-color: rgba(255, 255, 255, 0.9);
+  }
+
+  &__burger-checkbox:checked ~ .nav__list {
+    display: flex;
+  }
+
+  &__link {
+    position: relative;
+    max-width: 150px;
+    color: $color-main-accent;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: all 0.3s;
+    
+    &:hover,
+    &.is-active {
+      color: $color-moyo-blue;
+    }
+
+    &::after,
+    &.is-active::after {
+      content: "";
+      position: absolute;
+      bottom: 0;
+      display: block;
+      height: $dim-menu-underscore;
+      width: 100%;
+      border-radius: $dim-menu-underscore;
+      background-color: $color-moyo-blue;
+      opacity: 0;
+      transition: all 0.3s;
+    }
+
+    &:hover::after,
+    &.is-active::after {
+      opacity: 1;
+    }
+  }
+
+  @media screen and (min-width: 768px) {
+
+    // placeholders for media
+    %flex-center1 {
```

Comment:
I'm not a fun of using names like `%flex-center1`, it will become difficult to read and maintain in a bigger project.